mage_ai/api/resources/UserResource.py

```diff
@@ -126,18 +120,13 @@ def update(self, payload, **kwargs):
             role_ids = payload.get('roles_new', [])
             roles_new = self.check_roles(role_ids)
 
-            missing_values = []
-            if len(roles_new) == 0:
-                missing_values.append('roles')
-
             payload['roles_new'] = roles_new
 
-            if len(missing_values) >= 1:
-                error.update(
-                    {'message': 'Missing required values: {}.'.format(', '.join(missing_values))})
-                raise ApiError(error)
-
-            access = get_access_for_roles(roles_new, Permission.Entity.PROJECT, get_repo_path())
+            access = get_access_for_roles(
+                roles_new,
+                Permission.Entity.PROJECT,
+                get_project_uuid(),
```

Comment:
Does it work for project that has no uuid?

Reply:
every project should have a uuid after this PR. also there shouldn't be any project level permissions created before this unless a user manually created one.